packages/longhorn-1.1/generated-changes/patch/Chart.yaml.patch

```diff
@@ -13,3 +13,7 @@
 +  catalog.cattle.io/display-name: "Longhorn"
```

Comment:
Diff seems to show:

```diff
diff -uNr charts/longhorn-1.1/longhorn/1.1.200+up1.1.2/Chart.yaml charts/longhorn-1.1/longhorn/1.1.300-rc3+up1.1.3-rc3/Chart.yaml
--- charts/longhorn-1.1/longhorn/1.1.200+up1.1.2/Chart.yaml	2021-12-09 12:37:14.000000000 -0800
+++ charts/longhorn-1.1/longhorn/1.1.300-rc3+up1.1.3-rc3/Chart.yaml	2021-12-14 09:45:58.000000000 -0800
@@ -2,14 +2,16 @@
   catalog.cattle.io/auto-install: longhorn-crd=match
   catalog.cattle.io/certified: rancher
   catalog.cattle.io/display-name: Longhorn
+  catalog.cattle.io/kube-version: '>= v1.16.0-0, < v1.22.0-0'
   catalog.cattle.io/namespace: longhorn-system
-  catalog.cattle.io/rancher-version: <2.5.99-0
   catalog.cattle.io/os: linux
   catalog.cattle.io/provides-gvr: longhorn.io/v1beta1
+  catalog.cattle.io/rancher-version: <2.5.99-0
   catalog.cattle.io/release-name: longhorn
   catalog.cattle.io/ui-component: longhorn
+  catalog.cattle.io/upstream-version: 1.1.3-rc3
 apiVersion: v1
-appVersion: v1.1.2
+appVersion: v1.1.3-rc3
 description: Longhorn is a distributed block storage system for Kubernetes.
 home: https://github.com/longhorn/longhorn
 icon: https://raw.githubusercontent.com/cncf/artwork/master/projects/longhorn/icon/color/longhorn-icon-color.png
@@ -21,7 +23,7 @@
 - device
 - iscsi
 - nfs
-kubeVersion: '>=v1.16.0-r0'
+kubeVersion: '>= v1.16.0-0, < v1.22.0-0'
 maintainers:
 - email: maintainers@longhorn.io
   name: Longhorn maintainers
@@ -36,4 +38,4 @@
 - https://github.com/longhorn/longhorn-ui
 - https://github.com/longhorn/longhorn-tests
 - https://github.com/longhorn/backing-image-manager
-version: 1.1.200+up1.1.2
+version: 1.1.300-rc3+up1.1.3-rc3
```

w.r.t. setting the **Helm kubeVersion**, @innobead just to confirm; this particular chart version is **never** expected to be deployed (even in an officially unsupported fashion) within a k8s 1.15- or 1.22+ cluster?

If narrowing the scope of that kubeVersion is something that the longhorn team would ever want to do in the future, we should leave that constraint in the Rancher kube-version and drop it from the Helm kubeVersion flag. If not, this looks good as-is.

cc: @brendarearden

Reply:
That makes sense to me; I'm expecting that to be the case, which is why I approved regardless

Reply:
Yeah, Longhorn v1.1.3 only supports Kubernetes >= 1.16 and < 1.22 (not including 1.22)

Reply:
> If narrowing the scope of that kubeVersion is something that the longhorn team would ever want to do in the future, we should leave that constraint in the Rancher kube-version and drop it from the Helm kubeVersion flag

@aiyengar2 Can you give more detail about which variable to set?

Reply:
I don't think any variable needs to be changed, based on your answer in https://github.com/rancher/charts/pull/1639#discussion_r768955985.

If there's no reason why we would ever want to make this **specific chart version** available to 1.15- or 1.22+, then the kubeVersion and kube-version can be the same.